functions/Get-DbaSpn.ps1

```diff
@@ -71,76 +68,81 @@ Returns a custom object with SearchTerm (ServerName) and the SPNs that were foun
 			}
 		}
 		
-		ForEach ($account in $AccountName)
+		if ($ShowAccountNameAll)
```

Comment:
Regions are not that useful at seperating logic in your code unless the editor that you use supports it. Which not every editor used by contributors is going to support it, so it is not really any different than just adding comments.